inquire/src/lib.rs

```diff
@@ -28,7 +30,7 @@
 //!   - Custom extensions for files created by [`Editor`] prompts;
 //!   - and many others!
 //!
-//! \* Date-related features are available by enabling the `date` feature.
+//! \* Date-related features are available by enabling the `date` feature and enable the `editor` feature for the text editor features
```

Comment:
```suggestion
```

This line was accidentally kept I believe

Reply:
Thanks removed